src/providers/wfs/qgswfsshareddata.cpp

```diff
@@ -27,6 +27,7 @@ QgsWFSSharedData::QgsWFSSharedData( const QString &uri )
   , mURI( uri )
 {
   mHideProgressDialog = mURI.hideDownloadProgressDialog();
+  mServerPrefersCoordinatesForTransactions_1_1 = uri.contains( QLatin1String( "/arcgis/" ), Qt::CaseSensitivity::CaseInsensitive );
```

Comment:
this is wrong and I'll remove this: it is now an option in the connection config.